### ISSUE TITLE
Conversion to Biopython objects?

### ISSUE THREAD

**Issue by chiragmatkar** [alito/CID-miRNA#1]
Hi,

I like the work you have done.I was working on annotation of Homo sapiens Hypothetical proteins by the help of miRNAs once.You can extend the sequences (example input fasta) to biopython seq objects for easy retrieval of seq information using ids etc for the web server as well.

Thanks,
Chirag

**Comment by alito** [comment]
Yes, I should look into using biopython at some point. Thanks for the tip!

**Comment by chiragmatkar** [comment]
You are welcome.Something to start on with:
https://github.com/chiragmatkar/Bioinformatics/blob/master/miRNA/miRNA_Parser.py

On Sat, Jun 20, 2015 at 4:31 PM, Alejandro Dubrovsky <
notifications@github.com> wrote:

> Yes, I should look into using biopython at some point. Thanks for the tip!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alito/CID-miRNA/issues/1#issuecomment-113746296>.
>



-- 

Thanks and Regards,
Chirag Matkar
